tfprotov5/tf5server/server.go

```diff
@@ -1202,3 +1202,56 @@ func invalidDeferredResponseDiag(reason tfprotov5.DeferredReason) *tfprotov5.Dia
 			fmt.Sprintf("Deferred reason - %q", reason.String()),
 	}
 }
+
+func (s *server) ValidateListResourceConfig(ctx context.Context, protoReq *tfplugin5.ValidateListResourceConfig_Request) (*tfplugin5.ValidateListResourceConfig_Response, error) {
+	rpc := "ValidateListResourceConfig"
+	ctx = s.loggingContext(ctx)
+	ctx = logging.RpcContext(ctx, rpc)
+	ctx = logging.ResourceContext(ctx, protoReq.TypeName)
```

Comment:
Let's define a new `logging.ListResourceContext` method, for consistency with `DataSourceContext` and `EphemeralResourceContext`.